ansible/ansible-ai-connect-service
# AAP-26972: One click trial - Adjust the Terms and Conditions URL

Jira Issue: <https://issues.redhat.com/browse/AAP-26972>

## Description
- Updating the trial home page `terms and conditions` link to the following URL: `https://www.ibm.com/terms/?id=i126-9853`
- Opening the URL in a new tab, instead of staying on same page (you have to come back later)
- I understand at this point no one uses the /trial/terms.html template and its django view class, so dropping as well

![Screenshot from 2024-08-16 21-52-19](https://github.com/user-attachments/assets/1a87abf9-fdb5-4cf7-8910-f8ab962d02cb)

## Testing

1. Pull down the PR
2. Login into the service
3. Navigate to the `/trial` page, and see the link

### Scenarios tested
Local

## Production deployment
